test/lib/cli_test.rb

```diff
@@ -31,6 +31,7 @@ module ForemanMaintain
             plugin                        Manage optional plugins on your server
             self-upgrade                  Perform major version self upgrade
             maintenance-mode              Control maintenance-mode for application
+            report                        Generate the usage reports
```

Comment:
```suggestion
            report                        Generate usage report
```

you updated the string on my suggestion, but not the test

Reply:
https://github.com/ares/foreman_maintain/pull/15